dev/merge_spark_pr.py

```diff
@@ -39,6 +39,9 @@
 except ImportError:
     JIRA_IMPORTED = False
 
+if sys.version_info[0] >= 3:
+    raw_input = input
```

Comment:
Does this script run with Python 3+ now?

Reply:
It creates a new function called __raw_input()__ that is identical to the builtin __input()__.